tmail-backend/apps/distributed/src/test/java/com/linagora/tmail/james/app/SchedulerReconnectionHandlerIntegrationTest.java

```diff
@@ -69,6 +70,7 @@ class DeletedMessageVaultQueueEnabled {
                 .build())
             .server(configuration -> DistributedServer.createServer(configuration)
                 .overrideWith(new LinagoraTestJMAPServerModule())
+                .overrideWith(new OpenPaasModule())
```

Comment:
yesterday's attempt was wrong, i dropped the commit and simply added the `OpenPaasModule`, it resolved the problem